### ISSUE TITLE
gkr-pam: unable to locate daemon control file

### ISSUE THREAD

**Issue by slackernetuk** [Dlackware/gnome-systemd#26]
Few dyas ago I've built master on a fresh installed SlackwareCurrent VM. It seems we've there a issue with slacks pam.

Can you comfirm that? If yes, is there any workaround to solve it?

[log.txt](https://github.com/Dlackware/gnome/files/4680226/log.txt)

`

**Comment by belka-ew** [comment]
yes, I see it in my logs, but it doesn't seem to be fatal. But I should say my current system is not very "clean".
I spent so much time configuring PAM with GDM and I'm still overlooking something... Unfortunately I don't have a workaround, I have to investigate first.
